src/draw/lv_draw_vector.c

```diff
@@ -438,12 +476,12 @@ void lv_vector_path_append_path(lv_vector_path_t * path, const lv_vector_path_t
     uint32_t npoint_size = lv_array_length(&subpath->points);
 
     lv_array_resize(&path->ops, ops_size + nops_size);
-    uint8_t * start_ops = lv_array_get(&path->ops, ops_size - 1) + sizeof(lv_vector_path_op_t);
+    uint8_t * start_ops = path->ops.size == 0 ? path->ops.data : lv_array_get(&path->ops, ops_size - 1) + sizeof(lv_vector_path_op_t);
```

Comment:
It has been fixed here.
You can rebase to master now.
https://github.com/lvgl/lvgl/pull/5300/files#diff-1964caa407fbbf719ca3cd91996e37573dec510d6f48e88f8c22e1183fdc3915L433-R443